src/components/layout/Search/SearchArea.module.css

```diff
@@ -44,3 +44,20 @@
   padding: 7px 25px;
   border-radius: 10px;
 }
+
+.searchBtn:hover {
+  background-color: #dddcdc;
+}
+
+.searchBtn:active {
+  background-color: #fc0707;
```

Comment:
Change this active background color to `#d3d3d3`. The red color here was just for demonstration purposes so you can clearly see when the `:active` CSS state is triggered.